assets/src/apps/authoring/components/PropertyEditor/schemas/lesson.ts

```diff
@@ -1,25 +1,119 @@
+import CustomFieldTemplate from '../custom/CustomFieldTemplate';
 const lessonSchema = {
   type: 'object',
   properties: {
-    title: {
+    Size: {
+      type: "object",
+      title: "Dimensions",
+      properties: {
+        width: { type: 'number' },
+        height: { type: 'number' }
+      }
+    },
+    defaultTextStyle: {
       type: 'string',
-      options: { label: 'Title' },
+      title: 'Text Style',
+      enum: ['title', 'heading', 'sub-heading','body-text','small-text','caption','code'],
+      enumNames: ['Title', 'Heading', 'Sub-heading', 'Body Text', 'Small Text', 'Caption', 'Code']
+    },
+    Appearance: {
+      type: 'object',
+      title: 'Lesson Appearance',
+      properties: {
+        theme: {
+          type:'string',
+          title: 'Lesson Theme',
+          enum: [
+            'Light Responsive', 'Blue Responsive','Dark Responsive','Material Responsive', 'Light', 'Dark'
+          ]
+        },
+        customCssUrl: {
+          type:'string',
+          title: 'Custom CSS URL'
+        },
+        headingStyleType: {
+          type:'string',
+          title: 'Lesson Title Style',
+          enum: ['Title', 'Heading', 'Sub-heading']
+        },
+        renderAccessibleOrderedList: {
```

Comment:
some of these we don't actually use in delivery, so we don't need to be able to author them now

Reply:
ohkk.... so exactly which one i should remove?

Reply:
we don't use "headingStyleType", "renderAccessibleOrderedList", "showOptionsMenu", "enableLearningApps", "globalNumberFormat", "advancedAuthoring", or "defaultTextStyle"